File: pacote-download/Iniciante/JavaScript/Uri1049-Animal.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n')

var vertebradoOuInvertebrado = String(lines.shift())
var especie = String(lines.shift())
var caracteristica = String(lines.shift())

//vertebrado
if (vertebradoOuInvertebrado === 'vertebrado' && especie === 'ave' && caracteristica === 'carnivoro'){
    console.log('aguia')
}
if (vertebradoOuInvertebrado === 'vertebrado' && especie === 'ave' && caracteristica === 'onivoro'){
    console.log('pomba')
}
if (vertebradoOuInvertebrado === 'vertebrado' && especie === 'mamifero' && caracteristica === 'onivoro'){
    console.log('homem')
}
if (vertebradoOuInvertebrado === 'vertebrado' && especie === 'mamifero' && caracteristica === 'herbivoro'){
    console.log('vaca')
}

//invertebrado
if (vertebradoOuInvertebrado === 'invertebrado' && especie === 'inseto' && caracteristica === 'hematofago'){
    console.log('pulga')
}
if (vertebradoOuInvertebrado === 'invertebrado' && especie === 'inseto' && caracteristica === 'herbivoro'){
    console.log('lagarta')
}
if (vertebradoOuInvertebrado === 'invertebrado' && especie === 'anelideo' && caracteristica === 'hematofago'){
    console.log('sanguessuga')
}
if (vertebradoOuInvertebrado === 'invertebrado' && especie === 'anelideo' && caracteristica === 'onivoro'){
    console.log('minhoca')
}

File: pacote-download/Iniciante/Python/Uri1049-Animal.py
vertebradoOuInvertebrado = str(input().strip().lower())
especie = str(input().strip().lower())
caracteristica = str(input().strip().lower())

#vertebrado
if vertebradoOuInvertebrado == 'vertebrado' and especie == 'ave' and caracteristica == 'carnivoro':
    print('aguia')
elif vertebradoOuInvertebrado == 'vertebrado' and especie == 'ave' and caracteristica == 'onivoro':
    print('pomba')
elif vertebradoOuInvertebrado == 'vertebrado' and especie == 'mamifero' and caracteristica == 'onivoro':
    print('homem')
elif vertebradoOuInvertebrado == 'vertebrado' and especie == 'mamifero' and caracteristica == 'herbivoro':
    print('vaca')

#invertebrado
if vertebradoOuInvertebrado == 'invertebrado' and especie == 'inseto' and caracteristica == 'hematofago':
    print('pulga')
elif vertebradoOuInvertebrado == 'invertebrado' and especie == 'inseto' and caracteristica == 'herbivoro':
    print('lagarta')
if vertebradoOuInvertebrado == 'invertebrado' and especie == 'anelideo' and caracteristica == 'hematofago':
    print('sanguessuga')
elif vertebradoOuInvertebrado == 'invertebrado' and especie == 'anelideo' and caracteristica == 'onivoro':
    print('minhoca')


